data/entities.yaml

```diff
@@ -81,6 +90,14 @@
     be found by drilling beneath the surface.
   properties: [portable]
 
+- name: copper vein
```

Comment:
I was kind of imagining that the copper and iron veins would be *inside* the mountain tunnels, i.e. first you have to drill into the mountains and once you're in there you can find some veins, and it's not visually obvious where you have to drill to find the copper.

One idea would be something like this:
1. Change `copper vein` to look the same as a `mountain` and be `unwalkable`.  In other words a `copper vein` is now a special kind of mountain which has copper inside.
2. `drill`ing a `copper vein` produces a `copper mine`, which looks the same as a `mountain tunnel` but can be drilled to get copper.

What do you think?